.scripts/mark_file/main.py

```diff
@@ -0,0 +1,111 @@
+import polib
+import os
+import datetime
+import requests
+
+
+def entry_check(pofile):
+    lines_tranlated = len(pofile.translated_entries())
+    lines_untranlated = len(pofile.untranslated_entries())
+
+    if lines_tranlated == 0:
+        result = "❌"
+    elif lines_untranlated == 0:
+        result = "✅"
+    else:
+        lines_all = lines_tranlated + lines_untranlated
+        progress = lines_tranlated / lines_all
+        progress = round(progress*100, 2)
+        result = f"Ongoing, {str(progress)} %"
```

Comment:
- 一個好習慣是 var name 不要有 overwriting
- f-string 內的 var 會被轉換成 string，不用自己轉
```suggestion
        progress_percentage = round(progress*100, 2)
        result = f"Ongoing, {progress_percentage}%"
```